vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -138,6 +138,24 @@ instances:
     #   cluster:
     #     - <CLUSTER_REGEX>            # Only possible with "collection_type: historical"
 
+    ## @param collect_tags - boolean - optional - default: false
+    ## If true, the vSphere Tags will be collected.
+    ## The final tag sent to Datadog is composed of the vSphere tag category as key and vSphere tag name as value.
+    ##    <TAG_CATEGORY>:<TAG_NAME>
+    ## Examples:
+    ## - os_type:windows
+    ## - application_name:my_app
+    #
+    # collect_tags: false
+
+    ## @param vsphere_tags_prefix - boolean - optional - default: vsphere_
+    ## Tags coming from vSphere will be prefixed with this prefix.
+    ## Examples with the default `vsphere_` prefix:
+    ## - vsphere_os_type:windows            # where `os_type` is the category name and `windows` the tag name
+    ## - vsphere_application_name:my_app    # where `application_name` is the category name and `my_app` the tag name
+    #
+    # vsphere_tags_prefix: vsphere_
```

Comment:
Actually what do you think of having the possibility to use a prefix but defaulting to None? I think it's a better experience to have tags on datadog that match what you have on vSphere. But as you said, that can lead to some tags duplication so having the ability to add a custom prefix is great.

Also, note that most default tags start with vsphere (`vsphere_type`, `vsphere_host`, `vsphere_datastore`, `vsphere_folder` ....), it will probably cause more collision to default to vsphere.

Reply:
> Actually what do you think of having the possibility to use a prefix but defaulting to None? I think it's a better experience to have tags on datadog that match what you have on vSphere.

Yes, make sense. And if the customer have some conflicting or choose to, they can still had a prefix.